main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    car
+//  action 1:    car drives forward
+car.forward();
+//  action 2:    car drives backwards
+car.backwards();
+//  action 3:    car stops
+car.stops();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    cat
+//  action 1:    cat eats
+cat.eat();
+//  action 2:    cat drinks
+cat.drink();
+//  action 3:    cat poops
+cat.poop();
+//  3)    
+//  Noun:    heatPump
+//  action 1:    heat pump blows cool air
+heatPump.warmAir();
+//  action 2:    heat pump blows warm air
+heatPump.coldAir();
+//  action 3:    heat pump makes filter dirty
+heatPump.dirtyFilter();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    refrigerator
+//  action 1:    refrigerator keeps food cold
+refrigerator.coldFood();
+//  action 2:    refrigerator makes ice
+refrigerator.makeIce();
+//  action 3:    refrigerator keeps food frozen
+refrigerator.frozenFood();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    spaceship
+//  action 1:    spaceship shoots phasers
+spaceship.shootPhasers();
+//  action 2:    spaceship goes to lightspeed
+spaceship.goToLightspeed();
+//  action 3:    spaceship cloaks itself
+spaceship.cloak();
+//  6)    
+//  Noun:    bird
+//  action 1:    bird flys
+bird.fly();
+//  action 2:    bird eats bugs
+bird.eatBugs();
+//  action 3:    bird builds nest
+bird.buildNest();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    dog
+//  action 1:    dog bites mailman
+dog.biteMailman();
+//  action 2:    dog chases cat
+dog.chaseCat;
+//  action 3:    dog eats kibble
+dog.eatKibble();
+//  8)   
+//  Noun:    flower
+//  action 1:    flower blooms
+flower.bloom();
+//  action 2:    flower gives off fragrance
+flower.fragrance();
+//  action 3:    flower wilts
+flower.wilt();
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    cellPhone
+//  action 1:    cell phone rings
+cellPhone.ring();
+//  action 2:    cell phone battery discharges
+cellPhone.batteryDischarge();

```

Comment:
the noun here is `cell phone battery`, not `cell phone`.  those are two completely different things.
change the action.
- [ ] revise